leakcanary-android-core/src/main/java/leakcanary/LeakCanary.kt

```diff
@@ -145,6 +146,10 @@ object LeakCanary {
    */
   @Volatile
   var config: Config = if (AppWatcher.isInstalled) Config() else InternalLeakCanary.noInstallConfig
+    set(value) {
+      field = value
+      SharkLog.d { "Updated LeakCanary.config to ${AppWatcher.config} " }
```

Comment:
Nit: `$value` should work too?

Reply:
Not to mention that the code is actually wrong here